src/components/ToolMenu/LayerTree/LayerTreeContextMenu/index.tsx

```diff
@@ -139,6 +159,28 @@ export const LayerTreeContextMenu: React.FC<LayerTreeContextMenuProps> = ({
     });
   };
 
+  const downloadLayer = (url: string) => {
+    if (!layer) {
+      return;
+    }
+    const reqOpts = {
+      method: 'GET',
+      headers: {
+        ...getBearerTokenHeader(client?.getKeycloak())
```

Comment:
This should be applied to layers that are configured to do so only.

Reply:
yes, it is fixed in 7e07cc76c5179b2f01be452d57a32fe7458148d9